src/signals/incident-management/containers/IncidentDetail/constants.ts

```diff
@@ -38,3 +38,9 @@ export const PREVIEW =
 export const EDIT = 'sia/incidentManagement/containers/IncidentDetail/EDIT'
 export const EXTERNAL =
   'sia/incidentManagement/containers/IncidentDetail/EXTERNAL'
+
+export const DELETE_CHILD = 'sia_delete_attachment_of_child_signal'
+export const DELETE_NORMAL = 'sia_delete_attachment_of_normal_signal'
+export const DELETE_OTHER = 'sia_delete_attachment_of_other_user'
+export const DELETE_PARENT = 'sia_delete_attachment_of_parent_signal'
+export const SIA_WRITE = 'sia_write'
```

Comment:
This is not the correct permission. It also should not be 1 but 2 permissions that must be checked.

Reply:
I've incorporated the new permissions.